puppet/zulip/templates/postgresql/9.5/postgresql.conf.template.erb

```diff
@@ -549,11 +549,11 @@ timezone = 'localtime'
 					# encoding
 
 # These settings are initialized by initdb, but they can be changed.
-lc_messages = 'en_US.UTF-8'			# locale for system error message
+lc_messages = 'C.UTF-8'			# locale for system error message
 					# strings
-lc_monetary = 'en_US.UTF-8'			# locale for monetary formatting
-lc_numeric = 'en_US.UTF-8'			# locale for number formatting
-lc_time = 'en_US.UTF-8'				# locale for time formatting
+lc_monetary = 'C.UTF-8'			# locale for monetary formatting
+lc_numeric = 'C.UTF-8'			# locale for number formatting
+lc_time = 'C.UTF-8'				# locale for time formatting
```

Comment:
Hmm, what is the impact of changing the postgres locale?  (This section was from the postgres defaults, not something we set directly).  

Both in terms of what those settings control (which I hopefully isn't relevant for anything but `manage.py dbshell`) as well as whether changing this is any more disruptive than a server restart.

Reply:
This actually makes it _more_ consistent across versions.  Before this change:
```
puppet/zulip/templates/postgresql/10/postgresql.conf.centos.template.erb
580:#lc_messages = 'C'			# locale for system error message
582:#lc_monetary = 'C'			# locale for monetary formatting
583:#lc_numeric = 'C'			# locale for number formatting
584:#lc_time = 'C'				# locale for time formatting

puppet/zulip/templates/postgresql/10/postgresql.conf.template.erb
580:lc_messages = 'C.UTF-8'			# locale for system error message
582:lc_monetary = 'C.UTF-8'			# locale for monetary formatting
583:lc_numeric = 'C.UTF-8'			# locale for number formatting
584:lc_time = 'C.UTF-8'				# locale for time formatting

puppet/zulip/templates/postgresql/11/postgresql.conf.template.erb
580:lc_messages = 'C.UTF-8'			# locale for system error message
582:lc_monetary = 'C.UTF-8'			# locale for monetary formatting
583:lc_numeric = 'C.UTF-8'			# locale for number formatting
584:lc_time = 'C.UTF-8'				# locale for time formatting

puppet/zulip/templates/postgresql/12/postgresql.conf.template.erb
665:lc_messages = 'C.UTF-8'			# locale for system error message
667:lc_monetary = 'C.UTF-8'			# locale for monetary formatting
668:lc_numeric = 'C.UTF-8'			# locale for number formatting
669:lc_time = 'C.UTF-8'				# locale for time formatting

puppet/zulip/templates/postgresql/13/postgresql.conf.template.erb
694:lc_messages = 'en_US.UTF-8'			# locale for system error message
696:lc_monetary = 'en_US.UTF-8'			# locale for monetary formatting
697:lc_numeric = 'en_US.UTF-8'			# locale for number formatting
698:lc_time = 'en_US.UTF-8'				# locale for time formatting

puppet/zulip/templates/postgresql/9.3/postgresql.conf.template.erb
495:lc_messages = 'en_US.UTF-8'			# locale for system error message
497:lc_monetary = 'en_US.UTF-8'			# locale for monetary formatting
498:lc_numeric = 'en_US.UTF-8'			# locale for number formatting
499:lc_time = 'en_US.UTF-8'				# locale for time formatting

puppet/zulip/templates/postgresql/9.5/postgresql.conf.template.erb
552:lc_messages = 'en_US.UTF-8'			# locale for system error message
554:lc_monetary = 'en_US.UTF-8'			# locale for monetary formatting
555:lc_numeric = 'en_US.UTF-8'			# locale for number formatting
556:lc_time = 'en_US.UTF-8'				# locale for time formatting

puppet/zulip/templates/postgresql/9.6/postgresql.conf.template.erb
568:lc_messages = 'en_US.UTF-8'			# locale for system error message
570:lc_monetary = 'en_US.UTF-8'			# locale for monetary formatting
571:lc_numeric = 'en_US.UTF-8'			# locale for number formatting
572:lc_time = 'en_US.UTF-8'				# locale for time formatting
```

Given that we've been running with `C.UTF-8` for all of the PostgreSQL 12 databases for the last while, I think it's likely to be Fine for new ones.  It _will_ force a database restart, but the effects will be potentially changes to error messages in the logs; I believe the client's locale may be what affects `manage.py dbshell`.

Reply:
Yeah, that makes sense.  Since postgres 12 was our previous default, we can feel pretty confident in this being safe.